src/main/java/lotto/view/OutputView.java

```diff
@@ -0,0 +1,80 @@
+package lotto.view;
+
+import lotto.domain.LottoResult;
+import lotto.domain.LottoTicket;
+import lotto.domain.UserLottoTickets;
+import lotto.domain.UserLottoTicketsDto;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class OutputView {
+
+    private static final String ASKING_MONEY_MESSAGE = "구입금액을 입력해 주세요.";
+    private static final String ASKING_MANUAL_LOTTO_COUNT_MESSAGE = "수동으로 구매할 로또 수를 입력해 주세요.";
+    private static final String ASKING_MANUAL_LOTTO_NUMBER_MESSAGE = "수동으로 구매할 로또 번호를를 입력해 주세요.";
+    private static final String USER_LOTTE_COUNT_MESSAGE = "수동으로 %d장, 자동으로 %d장을 구매했습니다.\n";
+
+    private static final String ASKING_WINNING_LOTTO_NUMBER_MESSAGE = "지난 주 당첨 번호를 입력해주세요.";
+    private static final String ASKING_BONUS_NUMBER_MESSAGE = "보너스 볼을 입력해 주세요.";
+
+    private static final String FIFTH_LOTTO_MESSAGE = "3개 일치 (" + LottoResult.FIFTH_LOTTO_PRICE + "원)- ";
+    private static final String FORTH_LOTTO_MESSAGE = "4개 일치 (" + LottoResult.FORTH_LOTTO_PRICE + "원)- ";
+    private static final String THIRD_LOTTO_MESSAGE = "5개 일치 (" + LottoResult.THIRD_LOTTO_PRICE + "원)- ";
+    private static final String SECOND_LOTTO_MESSAGE = "5개 일치, 보너스 볼 일 (" + LottoResult.SECOND_LOTTO_PRICE + "원)- ";
+    private static final String FIRST_LOTTO_MESSAGE = "6개 일치 (" + LottoResult.FIRST_LOTTO_PRICE + "원)- ";
+
+    private static final String INCOME_RATE_FORMAT_MESSAGE = "총 수익률은 %f입니다.\n";
+
+
+    public static void askUserMoney() {
+        System.out.println(ASKING_MONEY_MESSAGE);
+    }
+
+    public static void askManualLottoCount() { System.out.println(ASKING_MANUAL_LOTTO_COUNT_MESSAGE);}
+
+    public static void askManualLottoNumber() {
+        System.out.println(ASKING_MANUAL_LOTTO_NUMBER_MESSAGE);
+    }
+
+    public static void printUserLottoTicket(int manualLottoCount, int automaticLottoCount,
+                                            UserLottoTicketsDto userLottoTicketsDto) {
+        System.out.printf(USER_LOTTE_COUNT_MESSAGE, manualLottoCount, automaticLottoCount);
+        for (LottoTicket ticket : userLottoTicketsDto.getLottoTickets()) {
+            printLottoTicket(ticket.getNumbers());
+        }
+    }
+
+    public static void printLottoTicket(List<Integer> numbers) {
+        System.out.println("[" + numbers.stream().map(String::valueOf).collect(Collectors.joining(",")) + "]");
```

Comment:
Stream을 잘 사용하셨군요!
다만 Stream을 사용할 경우엔 가독성을 위해 점(.)을 기준으로 줄바꿈을 하는 경우가 많습니다.
```suggestion
        String lottoTicket = numbers.stream()
                .map(String::valueOf)
                .collect(Collectors.joining(","));
        System.out.println("[" + lottoTicket + "]");
```

Java 8의 Stream의 활용에 대해서 더 궁금하시면 [모던 자바 인 액션](http://www.yes24.com/Product/Goods/77125987)을 읽어보세요.